ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/handlers/JsonRpcExecutorHandler.java

```diff
@@ -54,17 +54,23 @@ public class JsonRpcExecutorHandler {
   private static final Logger LOG = LoggerFactory.getLogger(JsonRpcExecutorHandler.class);
   private static final String SPAN_CONTEXT = "span_context";
   private static final String APPLICATION_JSON = "application/json";
-  private static final ObjectMapper JSON_OBJECT_MAPPER =
+  private static ObjectMapper jsonObjectMapper =
       new ObjectMapper()
           .registerModule(new Jdk8Module()); // Handle JDK8 Optionals (de)serialization
-  private static final ObjectWriter JSON_OBJECT_WRITER =
-      JSON_OBJECT_MAPPER
-          .writerWithDefaultPrettyPrinter()
-          .without(JsonGenerator.Feature.FLUSH_PASSED_TO_STREAM)
-          .with(JsonGenerator.Feature.AUTO_CLOSE_TARGET);
+  private static ObjectWriter jsonObjectWriter = createObjectWriter(jsonObjectMapper);
 
   private JsonRpcExecutorHandler() {}
 
+  public static Handler<RoutingContext> handler(
+      final ObjectMapper jsonObjectMapper,
+      final JsonRpcExecutor jsonRpcExecutor,
+      final Tracer tracer,
+      final JsonRpcConfiguration jsonRpcConfiguration) {
+    JsonRpcExecutorHandler.jsonObjectMapper = jsonObjectMapper;
+    JsonRpcExecutorHandler.jsonObjectWriter = createObjectWriter(jsonObjectMapper);
+    return handler(jsonRpcExecutor, tracer, jsonRpcConfiguration);
+  }
```

Comment:
rather than passing in a mapper with a predefined set of modules, what about just exposing a method to register modules with the already-existing mapper?

Reply:
This allows you to customize as you want and not just module